lab-carolina/lib/greet.js

```diff
@@ -0,0 +1,3 @@
+exports.greet = function(name) {
```

Comment:
You could easily have also just used `module.exports` directly, which would eliminate the need to call the method name when you require this module in elsewhere. 

Good work though!